Library/Formula/tesseract.rb

```diff
@@ -24,10 +25,21 @@ class Tesseract < Formula
   end
 
   option "all-languages", "Install recognition data for all languages"
+  option "with-training-tools", "Install OCR training tools"
+  option "with-opencl", "Enable OpenCL support"
 
   depends_on "libtiff" => :recommended
   depends_on "leptonica"
 
+  if build.with? "training-tools"
```

Comment:
If you want to use Tesseract with anything other than pre-built `.traineddata` etc. files, you need the training tools. I don't have a sense for the general need for that, but a lot of the discussion I see on [the Tesseract-OCR Google Group](https://groups.google.com/forum/#!forum/tesseract-ocr) is about custom OCR that needs training tools.

The other issue may be that due to the way some of the training tools use Pango/Cairo font rendering there seem to be a lot of issues with them on OS X vs. Linux. Part of why I'd like to have this as an option is so that people on OS X who *do* need to do OCR training can start to report those issues upstream in Tesseract and start getting them fixed.